commit 00bb4dcac7152bf71770fa94c25c5e304cfd09d2
Author: Carlos Garnacho <carlosg@gnome.org>
Date:   Mon Feb 24 16:23:25 2014 +0100

    libtracker-extract: define back extract[rules|modules]dir for Makefile.am to use
    
    Before commit 73c0decd, src/libtracker-extract/Makefile.am was receiving those
    indirectly from $(topsrcdir)/Makefile.decls. After that file was no longer included
    those remained empty, causing wrong file lookups in the built
    tracker-miner-fs and tracker-extract.
    
    So the declarations have been moved to a separate $(topsrcdir)/Makefile.extractdirs,
    so they're defined once and available to both src/libtracker-extract/Makefile.am and
    src/tracker-extract/Makefile.am.

diff --git a/Makefile.am b/Makefile.am
index 3b2b2eb..cb05c9b 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -73,6 +73,7 @@ functional-test:
 
 EXTRA_DIST +=                                          \
 	ChangeLog.pre-0-6-93                           \
+	Makefile.extractdirs                           \
 	gitlog-to-changelog                            \
 	intltool-extract.in                            \
 	intltool-merge.in                              \
diff --git a/Makefile.extractdirs b/Makefile.extractdirs
new file mode 100644
index 0000000..6e8b0a5
--- /dev/null
+++ b/Makefile.extractdirs
@@ -0,0 +1,2 @@
+extractrulesdir = $(datadir)/tracker/extract-rules
+extractmodulesdir = $(libdir)/tracker-$(TRACKER_API_VERSION)/extract-modules
diff --git a/src/libtracker-extract/Makefile.am b/src/libtracker-extract/Makefile.am
index b4bbaae..ad20135 100644
--- a/src/libtracker-extract/Makefile.am
+++ b/src/libtracker-extract/Makefile.am
@@ -1,3 +1,5 @@
+include $(top_srcdir)/Makefile.extractdirs
+
 AM_CPPFLAGS =                                          \
 	$(BUILD_CFLAGS)                                \
 	-I$(top_srcdir)/src                            \
diff --git a/src/tracker-extract/Makefile.am b/src/tracker-extract/Makefile.am
index 215971e..45c25f4 100644
--- a/src/tracker-extract/Makefile.am
+++ b/src/tracker-extract/Makefile.am
@@ -1,4 +1,4 @@
-extractrulesdir = $(datadir)/tracker/extract-rules
+include $(top_srcdir)/Makefile.extractdirs
 
 module_flags = -module -avoid-version -no-undefined
 rulesdir = $(extractrulesdir)
@@ -54,7 +54,6 @@ rules_files = \
 # NOTE:
 # We don't always link with libtracker-common, we only link
 # against it if we directly use functions in the .so
-extractmodulesdir = $(libdir)/tracker-$(TRACKER_API_VERSION)/extract-modules
 extractmodules_LTLIBRARIES = # Empty
 rules_DATA = # Empty
 
